docs/en/sql-reference/sql-statements/Administration/SHOW_PROCESSLIST.md

```diff
@@ -17,8 +17,8 @@ SHOW [FULL] PROCESSLIST
 ## Parameter
 
 | Parameter | Required | Description                                                                                                                      |
-| --------- | -------- |----------------------------------------------------------------------------------------------------------------------------------|
-| FULL      | No       | If you specify this parameter, the complete statement will be displayed. Otherwise, only display the first 100 characters of the statement |
+| --------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| FULL      | No       | If you specify this parameter, the complete SQL statement will be displayed. Otherwise, only the first 100 characters of the statement is displayed. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/c891f9c5-3b6b-4dbe-99a9-f74547ff7b69)
"is" should be changed to "are"